development/bin/ensure_release_notes.py

```diff
@@ -0,0 +1,89 @@
+"""Ensure that release notes exist for a given version."""
```

Comment:
Can you add some more docstring to document the rest of what this script is doing? Perhaps:

```suggestion
"""Ensure that release notes exist for a given version.

This script will do the following:
    Ensure a release notes file exists at `docs/admin/release_notes/version_{version}.md`.
    Ensure the `mkdocs.yml` file is updated to add the release notes file to the navigation.
    Ensure the `pyproject.toml` `tool.towncrier.filename` is updated to reference the release notes file.

It shouldn't be necessary to run this file manually. It is automatically called by `invoke generate-release-notes`.

Example:
    $ python ensure_release_notes.py --version '1.0.0'
"""
```